pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/manager/UnloadManager.java

```diff
@@ -38,10 +41,63 @@ public class UnloadManager implements StateChangeListener {
 
     private final UnloadCounter counter;
     private final Map<String, CompletableFuture<Void>> inFlightUnloadRequest;
+    private final String lookupServiceAddress;
+    private final LatencyMetric unloadLatency;
+    private final LatencyMetric assignLatency;
+    private final LatencyMetric releaseLatency;
 
-    public UnloadManager(UnloadCounter counter) {
+
+    private class LatencyMetric {
+
+        private static final long OP_TIMEOUT_NS = TimeUnit.HOURS.toNanos(1);
+        private static final double QUANTILES[] = {0.0, 0.50, 0.95, 0.99, 0.999, 0.9999, 1.0};
+        private static final String LABEL_NAMES[] = {"broker"};
```

Comment:
Should we label "bundleUnloading" for its metric name, similar to other unload metrics? or `broker` is required here?